.github/workflows/main.yml

```diff
@@ -67,11 +43,38 @@ jobs:
     - name: Run test with Node.js ${{ matrix.node-version }} and Karma
       run: |
         cd test
-        npm run coverage-ci
+        npm test
       env:
         CI: true
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
-      with:
-        file: ./test/coverage/lcov.info
-        fail_ci_if_error: true
+  # coverage:
```

Comment:
Please add comment here that coverage is being disabled in connection with this:
https://github.com/digitalbazaar/bedrock-karma/issues/14

Reply:
Added ef6f641.